src/strands/models/bedrock.py

```diff
@@ -89,6 +96,7 @@ class BedrockConfig(TypedDict, total=False):
         guardrail_redact_output_message: Optional[str]
         max_tokens: Optional[int]
         model_id: str
+        remove_tool_result_status: Optional[Literal["auto"] | bool]
```

Comment:
(nit, non-blocking) I think per updated standards:

```suggestion
        remove_tool_result_status: Literal["auto"] | bool | None = None
```